base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
This PR is turning into yet another black hole for core team members.  I fear these types of discussions are detracting from the generally positive spirit the team has maintained.  That kind of spirit is critical to a volunteer effort.

If it remains mired in side issues and contention, I would suggest closing the PR and moving on to more productive efforts.